modules/custom/az_news/config/install/core.entity_form_display.node.az_news.default.yml

```
@@ -85,15 +85,15 @@ content:
     type: text_textarea
     region: content
   field_az_byline:
-    weight: 1
+    weight: 2
     settings:
       size: 60
       placeholder: ''
     third_party_settings: {  }
     type: string_textfield
     region: content
   field_az_caption:
-    weight: 4
+    weight: 5
```

```suggestion
    weight: 5
```
Testing force commit